src/main/java/frc/robot/commands/WaitForCorrectShooterSpeed.java

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import java.util.function.DoubleSupplier;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.Shooter;
+
+public class WaitForCorrectShooterSpeed extends Command {
+  private final Shooter m_shooter;
+  private final DoubleSupplier m_leftRpm, m_rightRpm;
+  private static final double TOLERANCE = 5; // TODO Tune this later
```

Comment:
This is really low. Yes, we will tune it later, but we should start with a reasonable value. RPMs range up to 3000-5000, so what is a more sensible value?